plugins/services/src/js/structs/__tests__/Application-test.js

```diff
@@ -498,7 +498,7 @@ describe("Application", function() {
   });
 
   describe("#getWebURL", function() {
```

Comment:
I would say if we remove the method from the class we should also move the test into `Service-test.js` otherwise we kinda test the inheritance.

Reply:
`Service` struct doesnt has necessary implementation of `getLabels` method, we cant add tests there without extending it. But I removed these and added the single one from `Framework` instead.